spark-operator-api/src/main/java/org/apache/spark/k8s/operator/Constants.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.k8s.operator;
+
+public class Constants {
+  public static final String API_GROUP = "org.apache.spark";
+  public static final String API_VERSION = "v1alpha1";
+  public static final String LABEL_SPARK_APPLICATION_NAME = "spark.operator/spark-app-name";
+  public static final String LABEL_SPARK_OPERATOR_NAME = "spark.operator/name";
+  public static final String LABEL_SENTINEL_RESOURCE = "spark.operator/sentinel";
+  public static final String LABEL_RESOURCE_NAME = "app.kubernetes.io/name";
+  public static final String LABEL_SPARK_ROLE_NAME = "spark-role";
+  public static final String LABEL_SPARK_ROLE_DRIVER_VALUE = "driver";
+  public static final String LABEL_SPARK_ROLE_EXECUTOR_VALUE = "executor";
+  public static final String SENTINEL_RESOURCE_DUMMY_FIELD = "sentinel.dummy.number";
+
+  // Default state messages
+  public static final String DriverRequestedMessage = "Requested driver from resource scheduler.";
```

Comment:
Shall we use the same naming rule like `API_GROUP`? I guess all members of `Constants` should be in the same style.
```
- public static final String DriverRequestedMessage
+ public static final String DRIVER_REQUESTED_MESSAGE
```

I believe all members of `Constants` should be